db/schema.rb

```diff
@@ -10,7 +10,7 @@
 #
 # It's strongly recommended that you check this file into your version control system.
 
-ActiveRecord::Schema[7.0].define(version: 2022_10_12_173528) do
+ActiveRecord::Schema[7.0].define(version: 2022_10_31_203041) do
```

Comment:
conflict w/ another pr. not sure how to approach rectifying 2 different versions?

Reply:
pick the bigger number